pkg/guestagent/guestagent_linux.go

```diff
@@ -191,9 +191,8 @@ func (a *agent) collectEvent(ctx context.Context, st eventState) (*api.Event, ev
 
 func isEventEmpty(ev *api.Event) bool {
 	empty := &api.Event{}
-	copied := ev
```

Comment:
This regression seems caused in v0.21

https://github.com/lima-vm/lima/commit/72b7cb7fe2f8097eb363a64b557fd1258bba9f44#diff-4e96077f852132abc5fce986b8a80ec455c810b1db00d2cb23dbc8c4d2d9a365L182